src/inmanta/agent/cache.py

```diff
@@ -40,25 +40,31 @@ def __init__(
         timeout: float,
```

Comment:
If it was decided to keep 60 sec, don't we want this parameter to be optional? To me, it feels weird to have a parameter that will not be considered in some cases.